apps/client/src/common/components/view-params-editor/ViewParamsEditor.tsx

```diff
@@ -117,18 +117,30 @@ export default function ViewParamsEditor({ viewOptions }: EditFormDrawerProps) {
   const onParamsFormSubmit = (formEvent: FormEvent<HTMLFormElement>) => {
     formEvent.preventDefault();
 
-    const newParamsObject = Object.fromEntries(new FormData(formEvent.currentTarget));
+    const newParamsObject: Record<string, string> = {};
+
+    new FormData(formEvent.currentTarget).forEach((value, key) => {
+      // handle multi-select values by concating new values
+      if (key in newParamsObject) {
+        newParamsObject[key] = newParamsObject[key].concat(`,${value}`);
+      } else {
+        newParamsObject[key] = String(value);
+      }
+    });
+
     const newSearchParams = getURLSearchParamsFromObj(newParamsObject, viewOptions);
     setSearchParams(newSearchParams);
+
+    onClose();
   };
 
   return (
-    <Drawer isOpen={isOpen} placement='right' onClose={handleClose} variant='ontime' size='lg'>
-      <DrawerOverlay />
-      <DrawerContent>
-        <DrawerHeader>
-          <DrawerCloseButton size='lg' />
+    <DrawerRoot open={isOpen} placement='end' onOpenChange={handleClose} size='lg' trapFocus={false}>
+      <DrawerBackdrop zIndex={15} />
+      <DrawerContent positionerZIndex={15} portalled={false}>
```

Comment:
I worry that manual z-indexes are difficult to maintain. especially when they are not attached to an indexing system.

When i removed these properties everything seemed fine. Maybe it is safe to remove?